core/templates/external-js/tiddlywiki.js.tid

```diff
@@ -1,4 +1,4 @@
-title: $:/core/templates/tiddlywiki5.js
+title: $:/core/templates/twcore.js
```

Comment:
All these template name changes are not backwards compatible. Are they really needed?

Reply:
Hi @Jermolene, @pmario 

Restoring the original template name is not a problem. Are we keeping `twcore-<VERSION>.js` as the name of the downloaded core JS file? The filename was changed from `tiddlywiki5.js` to avoid confusion with node.js startup `tiddlywiki.js`.

Reply:
We have to keep the version in the core file, otherwise it's not possible to have 2 different files in one directory.